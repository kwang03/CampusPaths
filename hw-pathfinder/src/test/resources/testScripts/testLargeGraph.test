#Tests Dijkstra's on a large graph

CreateGraph myGraph

AddNode myGraph A
AddNode myGraph B
AddNode myGraph C
AddNode myGraph D
AddNode myGraph E
AddNode myGraph Z

AddEdge myGraph A B 4.0
AddEdge myGraph A C 2.0
AddEdge myGraph C B 1.0
AddEdge myGraph C D 8.0
AddEdge myGraph B D 5.0
AddEdge myGraph C E 10.0
AddEdge myGraph E D 2.0
AddEdge myGraph E Z 5.0
AddEdge myGraph D Z 6.0

FindPath myGraph A Z