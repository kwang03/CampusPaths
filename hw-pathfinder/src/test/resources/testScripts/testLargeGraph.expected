#Tests Dijkstra's on a large graph

created graph myGraph

added node A to myGraph
added node B to myGraph
added node C to myGraph
added node D to myGraph
added node E to myGraph
added node Z to myGraph

added edge 4.000 from A to B in myGraph
added edge 2.000 from A to C in myGraph
added edge 1.000 from C to B in myGraph
added edge 8.000 from C to D in myGraph
added edge 5.000 from B to D in myGraph
added edge 10.000 from C to E in myGraph
added edge 2.000 from E to D in myGraph
added edge 5.000 from E to Z in myGraph
added edge 6.000 from D to Z in myGraph

path from A to Z:
A to C with weight 2.000
C to B with weight 1.000
B to D with weight 5.000
D to Z with weight 6.000
total cost: 14.000